src/internal/kopia/backup_bases.go

```diff
@@ -55,56 +58,67 @@ func (bb *backupBases) SnapshotAssistBases() []ManifestEntry {
 		return nil
 	}
 
+	res := []ManifestEntry{}
+
+	for _, ab := range bb.assistBases {
```

Comment:
it's not much, but I count six locations where we do this same iteration.  Is that worth it's own func?

Reply:
this will be going away in the next PR. This is just to keep the interface intact so there's (relatively) less changes